mmdeploy/mmcv/cnn/__init__.py

```diff
@@ -1,5 +1,10 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .conv2d_adaptive_padding import (
```

Comment:
@SsTtOoNnEe  hi, could update the supported model config in the regression [yaml](https://github.com/open-mmlab/mmdeploy/blob/master/tests/regression/mmcls.yml)?